services/madoc-ts/translations/en/madoc.json

```diff
@@ -359,6 +367,10 @@
   "Log in": "Log in",
   "Login": "Login",
   "Logo": "Logo",
+  "Logo 1": "Logo 1",
+  "Logo 2": "Logo 2",
+  "Logo 3": "Logo 3",
+  "Logo 4": "Logo 4",
```

Comment:
There is no current way around this at the moment, but could look into it later (the auto-generated translations from the new form elements)